google/cloud/spanner/doc/spanner-main.dox

```diff
@@ -25,9 +25,9 @@ into your project.
 - [Client::Read()](@ref google::cloud::spanner::Client::Read()) - to read the
   rows in a table.
 - @ref spanner-error-handling to learn how the library reports run-time errors.
-- @ref spanner-env-logging enable logging. THis can be an effective approach to
-  diagnose many problems.
 - @ref spanner-env for environment variables affecting the library.
```

Comment:
This link leads to a page where a `:cloud:` becomes an image.

Reply:
Should be fixed in #11642